### PR TITLE
CI: backport single test for powered-off CirrOS VM

### PR DESCRIPTION
**Pull request by mrnold**
## Why the changes were made

Backport of #1439 to oadp-1.4. Partially fixes [OADP-4291](https://issues.redhat.com//browse/OADP-4291) for 1.4.

## How to test the changes made

make TEST_VIRT=true test-e2e

### CODE REVIEW

File: tests/e2e/virt_backup_restore_suite_test.go [hunk 1]
```diff
@@ -188,3 +230,45 @@ var _ = Describe("VM backup and restore tests", Ordered, func() {
...
+	accumulatedTestLogs = append(accumulatedTestLogs, describeRestore, restoreLogs)
+
+	if !brCase.SkipVerifyLogs {
+		Expect(restoreErrorLogs).Should(Equal([]string{}))
+	}
+
+	// Check if restore succeeded
+	succeeded, err := lib.IsRestoreCompletedSuccessfully(kubernetesClientForSuiteRun, dpaCR.Client, namespace, restoreName)
+	if brCase.RestoreErr != nil {
+		Expect(succeeded).To(Equal(false))
```
Comment (mateusoliveira43) [inline tests/e2e/virt_backup_restore_suite_test.go:256]: from CI logs, seems restore is working :eyes:

Comment (mateusoliveira43) [reply tests/e2e/virt_backup_restore_suite_test.go:256]: this PR was created after rebase, so it is using new openshift velero image. Could it be making tests pass?

Comment (mrnold) [reply tests/e2e/virt_backup_restore_suite_test.go:256]: I think that must be the case, I didn't realize it was fixed already.

Comment (mateusoliveira43) [reply tests/e2e/virt_backup_restore_suite_test.go]: just to make sure, this commit fixed the issue? https://github.com/openshift/velero/commit/7cb5b960190a7cb4cc490b3e5b0f72bf222a7f0f

Comment (mrnold) [reply tests/e2e/virt_backup_restore_suite_test.go]: I am pretty sure that's right, maybe @shubham-pampattiwar can confirm?

**Comment by mateusoliveira43** [comment]
/retest
infra

**Comment by kaovilai** [comment]
subscription suite test flake on apply update conflict.

/override ci/prow/4.16-e2e-test-aws

**Review by kaovilai — APPROVED** [review]

File: tests/e2e/virt_backup_restore_suite_test.go (hunk 1 above)
Comment (shubham-pampattiwar) [reply tests/e2e/virt_backup_restore_suite_test.go]: yes restored PVCs with pending state wont partially fail the restore

**Review by mateusoliveira43 — APPROVED** [review]
/lgtm

**Pull request merged**